tests/api.js

```diff
@@ -70,6 +73,10 @@ describe('Initialization', function() {
   });
 
   it('should detect missing cloudant', function(done) {
+    if (process.env.NOCK_OFF) {
+      this.skip();
```

Comment:
Cool, thanks for checking, I did think it only applied at the scope of the skip, but it would have been a nuisance to find otherwise later.